aws/provider.go

```diff
@@ -325,6 +325,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_autoscaling_policy":                           resourceAwsAutoscalingPolicy(),
 			"aws_autoscaling_schedule":                         resourceAwsAutoscalingSchedule(),
 			"aws_budgets_budget":                               resourceAwsBudgetsBudget(),
+			"aws_capacity_reservation":                         resourceAwsCapacityReservation(),
```

Comment:
Drive by note: We'll probably be looking for including `_ec2` in the resource name. 😄 Trying to get better about denoting the AWS service to prevent future collisions/confusion.

Reply:
OK - will update to `aws_ec2_capacity_reservaton` :-). Any plans to deprecate the existing non-prefixed resources?

Reply:
Thanks! We are not likely to deprecate them unless there's a _very_ good reason, but we could alias them. 🤔 Not sure if its worth the effort/confusion of switching the existing ones though, the naming request for new resources is more to prevent the problem from getting worse in the future.